### ISSUE TITLE
Suggestion: rename the TEST environment variable to something more specific like REVENUECAT_TEST

### ISSUE THREAD

**Issue by lapfelix** [RevenueCat/purchases-ios#492]
- [x] I have updated Purchases SDK to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/RevenueCat/purchases-ios/blob/develop/CONTRIBUTING.md)
- [x] I have read the [Help Center](https://support.revenuecat.com/hc/en-us)
- [x] I have read [docs.revenuecat.com](https://docs.revenuecat.com/)
- [x] I have searched for [existing Github issues](https://github.com/RevenueCat/purchases-ios/issues)

**Describe the bug**
Someone here couldn't build the app because they had `TEST` as an environment variable and Xcode was trying to fetch the test dependencies (ohhttpstubsand and nimble there was an error because ohhttpstubs doesn't specify a version):
```
Could not resolve package dependencies:
  because package purchases-ios is required using a version-based requirement and it depends on unversion package ohhttpstubs and root depends on purchases-ios 3.10.7, version solving failed.
```

Could the `TEST` environment variable be renamed to something more specific like REVENUECAT_TEST so the test dependencies aren't fetched unintentionally when Xcode resolves the package dependencies?

https://github.com/RevenueCat/purchases-ios/blob/develop/Package.swift#L7

Thank you!

1. Environment
   1. Platform: iOS
   2. SDK version: 3.10.7
   3. OS version: macOS 11.2
   4. Xcode version: 12.4
   5. How widespread is the issue. Percentage of devices affected.
2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
3. Steps to reproduce, with a description of expected vs. actual behavior
4. **Other information** (e.g. stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc.)

**Comment by lapfelix** [comment]
Thank you bot, this isn't exactly a bug but I just added the missing issue template stuff

**Comment by aboedo** [comment]
Hi @lapfelix!! Thanks for reporting. Don't mind the bot, you're good. 
We'll either rename or entirely remove if possible the variable. Ironically, the reason that the TEST variable was there in the first place was so that Xcode wouldn't fetch those dependencies unless someone really wanted to, but I guess we failed to consider the possibility that someone would be using the same variable for something else. 

I'll keep the ticket open until the fix is released.

**Comment by lapfelix** [comment]
Great, thank you!

**Comment by aboedo** [comment]
Hi! sorry for the delay here. 
The fix has been merged and released as a part of 3.12.0. 
Thanks again for reporting!
